src/coreio.i

```diff
@@ -0,0 +1,10 @@
+%module coreio
```

Comment:
wonder if we could isolate all files needed for `bitcoinpy` in a subdir? this apply basically for all the file type you added to `.gitignore` in 67f457b

Reply:
Yes we can and that would make a lot of sense, keeps the clutter down. Good point.

Reply:
Another question is whether the libs should be installable; should this be usable outside just testing a particular bitcoin C++ code base?

Installing the SWIG libraries creates another set of headaches, such as what happens if you have multiple bitcoind build dirs and want to make sure that you import locally with your python test code, instead of globally. So this is why I myself would rather avoid installing them for now.

It is possible right now - and should stay possible in the future IMO - to just open an `ipython` shell and play with the interfaces. Which I find to be helpful in both exploring the operation of current code as well as my own code for testing.

Reply:
> Another question is whether the libs should be installable

which libs?

Reply:
The SWIG modules. So that you could do things like

``` from bitcoinpy.primitives import uint256```

from python anywhere on your system.